docs/docs/reconciliation.md

```diff
@@ -20,14 +17,12 @@ Since an optimal algorithm is not tractable, we implement a non-optimal O(n) alg
 1. Two components of the same class will generate similar trees and two components of different classes will generate different trees.
 2. It is possible to provide a unique key for elements that is stable across different renders.
 
-In practice, these assumptions are ridiculously fast for almost all practical use cases.
-
+In practice, these assumptions are valid for almost all practical use cases.
 
 ## Pair-wise diff
 
 In order to do a tree diff, we first need to be able to diff two nodes. There are three different cases being handled.
 
-
 ### Different Node Types
 
```

Comment:
Can we use "element" instead of "node" here and below? While technically reconciliation also affects "nodes" (booleans and text), this article talks specifically about how we treat elements. It was written before modern terminology was introduced.